dev/camdenhells/REQUIREMENTS.md

```diff
@@ -178,7 +178,9 @@ Provide separate specification of a rendered view for a scene model. Each scene
   * Any number of frame renders to readable off-screen image buffers
 4. Multiple render targets per framebuffer (#1862)
   * E.g. for deferred lighting and ambient occlusion g-buffers
-  * Different size attachments are supported per render target
+  * All attachments per render target are the same size (resolution).
+  * Different storage formats per attachment are supported per render target.
```

Comment:
I'm guessing you mean "are supported per framebuffer", i.e. the attached color buffers can be of different internal formats?

Reply:
Correct